src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/AbstractReferenceFinder_GlobalSuppressions.cs

```diff
@@ -61,7 +61,7 @@ static bool SupportsGlobalSuppression(ISymbol symbol)
             return;
 
         // Check if we have any relevant global attributes in this document.
-        var info = await SyntaxTreeIndex.GetRequiredIndexAsync(state.Document, cancellationToken).ConfigureAwait(false);
+        var info = state.Cache.SyntaxTreeIndex;
```

Comment:
no need to fetch the index for a document.  we already are passing t the state object for that doc which has this info.